### ISSUE TITLE
Question : Integration with WebXR Virtual and Augmented Reality ?

### ISSUE THREAD

**Issue by arthurmougin** [WICG/webmonetization#136]
(Please close/remove if inappropriate)

[WebXR](https://github.com/immersive-web/webxr) allows the creation of an immersive session that allows a website to fully emerge a user inside a threejs or babylonjs environment (especially VR and headset AR). 
How does web monetization integrate with that immersion?
Does the user has to leave the immersive session to make a transaction?
Thank you in advance for your answer!

**Comment by adrianhopebailie** [comment]
Hey @arthurmougin ! Actually @anselm has done a bunch of experimentation in this area and I'm sure can share his experience

**Comment by anselm** [comment]
If you're hosting (or visiting) an immersive 3d site that has a coil payment pointer then of course it's just a background process of the coil plugin - should be the same capabilities as for 2d sites. That said for immersive VR gear such as the Oculus Quest some browsers don't support plugins at all. This makes it harder because those are the places where 3d shines, but unfortunately are also places where it is especially hard to sustainably create compelling content.

I ran into this issue when working at Mozilla on Web Monetization for Firefox Reality for the Oculus Quest. Coil was not whitelisted and there was a focus on user privacy. Looking just now I don't see it whitelisted still?

I did just receive a small amount of funding to start a new browser initiative - which I've just pasted initial thoughts on at https://orbitalweb.github.io . My take is that Coil and other solutions are very good, and we need to expose these in browsers as critical ecosystem fabric; so that creatives have more options to sustainably create the work that draws people to browsers in the first place - especially when competing against walled gardens (that do have healthy ecosystem models for artists).
